lib/core/typeinfo.nim

```diff
@@ -242,8 +248,7 @@ proc base*(x: Any): Any =
 proc isNil*(x: Any): bool =
   ## `isNil` for an any `x` that represents a sequence, string, cstring,
```

Comment:
remove from doc comment: sequence and string

Reply:
Oops, thanks!